hail/python/hail/methods/statgen.py

```diff
@@ -713,6 +713,10 @@ def linear_mixed_model(y,
     `mean_impute` to false if you expect no missing values, both for performance
     and as a sanity check.
 
+    In the example, if the markers constitute a small fraction of the original
```

Comment:
I think this should be a warning -- identical to what you have above. It's not clear to me what the example refers to.